website/docs/advanced/override_grammar/basic.md

```diff
@@ -126,7 +126,22 @@ To include a single quote in a single quoted string escape it : `\'`. Same for d
 </div>
 </div>
 
- 
+### Unquoted strings
+Hydra also allows unquoted strings to simplify the command line syntax.
+These strings can be made of alphanumeric characters, as well as the following symbols: `_/\+-.:$*`.
+Whitespaces are also allowed in-between these characters (but leading and trailing whitespaces are stripped).
+Finally, interpolations may be embedded within an unquoted string.
+
+Here are some examples showing how these may be used on the command line (note that here all
+quotes are "consumed" by the shell, so that from Hydra's point of view all these strings are unquoted):
+```bash
+python my_app.py url=https://hydra.cc
+python my_app.py url='https://${server}:${env:PORT}'
+python my_app.py msg='hello world'
+python my_app.py msg='  hello world  '  # same as above
+python my_app.py msg='hello^world'      # invalid, use a quoted '"hello^world"' instead
```

Comment:
I would prefer to train people to quote the whole override when quoting for the shell.
```suggestion
Override values can contain whitespaces and other special characters.
Some examples of legal values: `hello world`,`   hello world  `, `https://${server}:${env:PORT}`.
Unquoted whitespaces are allowed in the middle of the word, while leading and trailing whitespaces are stripped.
Normally, your shell will interpret values with whitesspaces as being multiple parameters (`key=a b` would be interpreted as `key=a` and `b`). To prevent this you can quote them with a single quote. For example:

```bash
$ python my_app.py 'msg=hello world'
```

Reply:
Taking your suggested commit -- will just follow-up with a small typo fix in it